notebooks/tutorials/cmle/sklearn_census.ipynb

```diff
@@ -0,0 +1,540 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Model Training and Prediction with Google Cloud Machine Learning Engine\n",
+    "In this notebook instance, you will use Cloud Machine Learning Engine to train a model using scikit-learn and serve the trained model. You will then use the served model to classify some new and unseen data.\n",
+    "\n",
+    "This notebook uses the [Census Data Set](https://archive.ics.uci.edu/ml/datasets/Census+Income) to demonstrate how to train a model on Cloud Machine Learning Engine (ML Engine).\n",
+    "\n",
+    "Before you jump in, let’s cover some of the different tools you’ll be using to get online prediction up and running on ML Engine. \n",
+    "\n",
+    "[Cloud ML Engine](https://cloud.google.com/ml-engine/) (CMLE) is a managed service that enables you to easily build machine learning models that work on any type of data, of any size.\n",
+    "\n",
+    "[Google Cloud Storage](https://cloud.google.com/storage/) (GCS) is a unified object storage for developers and enterprises, from live data serving to data analytics/ML to data archiving.\n",
+    "\n",
+    "[Cloud SDK](https://cloud.google.com/sdk/) is a command line tool which allows you to interact with Google Cloud products."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Step One: Setup the Environment\n",
+    "In order to use CMLE, first you need to [enable Cloud Machine Learning Engine and Compute Engine APIs](https://console.cloud.google.com/flows/enableapi?apiid=ml.googleapis.com,compute_component&_ga=2.217405014.1312742076.1516128282-1417583630.1516128282).\n",
+    "\n",
+    "You'll also need a number of environment variables to run this sample. We have already defined them for you:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get the current GCP Project ID:\n",
```

Comment:
I think it would be better to put placeholder values that the user has to change instead of showing complex logic.
Add something like this to the markdown cell above (remove "We have already defined them for you"):

"Replace the variables below with your Google Cloud project ID and a globally unique name for a new Cloud Storage bucket. For more information about naming buckets, see [Bucket name requirements](https://cloud.google.com/storage/docs/naming#requirements)."

Replace code cell with something like this:
```python
# Replace the string below with your project ID
project_id = "your-project-id"

# Replace the string below with a globally unique name for the new bucket
bucket_name = "your-new-bucket"
```

When I create the tests for this notebook, I can do the string replacements during execution. (ex. your-new-bucket => randomized-bucket23325453)